site/_posts/2020-08-11-advanced-registration-form.md

```diff
@@ -0,0 +1,176 @@
+
+> *Keep it short.* The mathematician Blaise Pascal famously said: “I have made this longer than usual because I have not had time to make it shorter.” This principle applies to web forms as well as prose writing. Eliminating unnecessary fields requires more time, but the reduced user effort and increased completion rates make it worthwhile. ... Every time you cut a field or question from a form, you increase its conversion rate — the business case for this guideline is that simple. 
+
+After you've examined the first step, create a second one. Add your custom fields to learn more about their house hunt.
+
+* Geographic area of interest
+* Minimum house search price
+* Maximum house search price
+
+After you've added these fields to the form, you'll see this:
+
+{% include _image.liquid src="/assets/img/blogs/advanced-registration-forms/form-second-step.png" alt="Adding fields to our second step." class="img-fluid" figure=false %}
+
+Feel free to rearrange the form fields within each step by clicking the arrows to move a field up or down. The form configuration specifies steps and field display order within those steps. If you need to move a field between steps, delete it from one step and add it to another. Each field can live in zero or one steps.
+
+To change field validation or other attributes, return to the "Fields" section to make your changes. When you're done tweaking the form to your liking, save it.
+
+## Specify the form for your application
+
+Now that you've created a form with custom fields, the next step is to specify which applications should use it. Forms and form fields can be reused in any application and any tenant. Since FusionAuth supports multiple tenants per instance, you can easily reuse the fields and forms you've created. For each application, simply assign the registration form to the application. Luckily, this is the next step for this tutorial.
+
+Navigate to the "Applications" tab and create a new FusionAuth application. As a reminder, an application is anything a user can sign into: a web application, native app, API, anything. In this case, you are building out the real estate application; pick something descriptive. How about "Real estate search"? 
+
+You must configure a redirect URL; this is where the user is sent when registration succeeds. To set this up, navigate to the "OAuth" tab of your application and enter "https://fusionauth.io" or another public website. For a production application, of course, you'd instead specify where a user should be sent after registration. Each application may have multiple redirect URLs. Configure where to send a user after successful registration by adding a `redirect_uri` to the registration form link. For this tutorial, you'll only add one and that will be automatically added to the registration URL when you are ready to test drive this from the user's perspective.
```

Comment:
This is actually the hand off for OAuth and the `redirect_uri` must call the token endpoint. While this is a necessary step, this paragraph doesn't tell the reader anything about OAuth. Perhaps we should link out to another blog or a documentation page that describes `redirect_uri` better? Or we should beef this paragraph up a bit to cover OAuth workflows and how registration is actually a nested flow that is part of the Authorization Code Grant.

Reply:
Hmmm. I wasn't able to find any docs about registration and the `redirect_uri`. I'm not sure about the value of talking about the token endpoint in this blogpost is, but I definitely feel like it should go into part of the guide. What am I missing?